lib/jgi_mg_assembly/utils/report.py

```diff
@@ -240,6 +339,12 @@ def _calc_m50_m90(self, stats_files, input_reads_count):
                     m90 = vals[2]
         return m50, m90
 
+    def _generate_graphics(self, covstats_file, output_dir):
```

Comment:
Maybe unused since you added that graphics module?

Reply:
Yes. Removed.